src/NServiceBus.Metrics/Reporters/NServiceBusMetricReport.cs

```diff
@@ -33,15 +33,10 @@ public void RunReport(MetricsData metricsData, Func<HealthStatus> healthStatus,
 
     async Task RunReportAsync(MetricsData metricsData)
     {
-        var serialized = JsonBuilderV2.BuildJson(metricsData);
-        var body = Encoding.UTF8.GetBytes(serialized);
-
-        var headers = new Dictionary<string, string>
-        {
-            [Headers.EnclosedMessageTypes] = "Metrics.Json.JsonMetricsContext, Metrics"
-        };
+        var stringBody = $@"{{""Data"" : {JsonBuilderV2.BuildJson(metricsData)}}}";
```

Comment:
Does JsonBuilderV2 only work on MetricsData? Because we could do:

```
JsonBuilderV2.BuildJson(new { Data = metricsData } });
```

Then we don't have to assemble json by hand